hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -394,6 +394,9 @@ public void close() throws IOException {
     transactionBuffer.close();
     HadoopExecutors.
         shutdown(installSnapshotExecutor, LOG, 5, TimeUnit.SECONDS);
+    scm.stop();
+    ExitUtils.terminate(1, "ScmStateMachine is closed, shutdown SCM",
+        StateMachine.LOG);
```

Comment:
`OzoneManager` has a `shutdown` method that performs these two steps.  I think it would be nice to make OM and SCM consistent by moving these to a new `shutdown` method in `StorageContainerManager`.

Reply:
Looks like `scm.stop` call also causes `StackOverflowError`:

https://github.com/apache/ozone/runs/4656696687#step:4:2870

Reply:
>I think it would be nice to make OM and SCM consistent by moving these to a new shutdown method in StorageContainerManager.

sure, will do this!